### DISCUSSION TITLE
No object found while running deconvolute

### DISCUSSION THREAD

**Discussion by andreselia** [omnideconv/immunedeconv#74] [Q&A] [answered]
Hi,
when running the deconvolute function, it appears that works well until an error occurred. 
My data is "matrix_data" (with genes in rows and sample ids in columns)
Here is my code and what obtained:

res = deconvolute(matrix_data, method = "quantiseq")

>>> Running quantiseq

Running quanTIseq deconvolution module

Gene expression normalization and re-annotation (arrays: FALSE)

Removing 17 noisy genes

Removing 15 genes with high expression in tumors

Signature genes found in data set: 135/138 (97.83%)

Mixture deconvolution (method: lsei)

Deconvolution sucessful!
Error in ---------------------------------------------------------------------------------------res = deconvolute(matrix_data,  : 
  object 'res' not found

**Comment by grst** [comment, marked as answer]
Answered in https://github.com/icbi-lab/immunedeconv/issues/75
